src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -150,7 +150,7 @@ public override void OnEventWritten(string name, object payload)
                         {
                             if (activity.IsAllDataRequested)
                             {
-                                activity.SetStatus(Status.Unset);
+                                activity.SetStatus(ActivityStatusCode.Unset);
```

Comment:
we dont need to set Unset. Its the default

Reply:
okay, now I'm not sure why the try/finally block was here. I removed it anyways. Thanks.